extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -163,4 +163,10 @@ export class AmpShareTracking extends AMP.BaseElement {
   }
 }
 
-AMP.registerElement('amp-share-tracking', AmpShareTracking);
+
+// Install the extension.
+// DO NOT SUBMIT: First, we must implement compiler rewrite path to unwrap
+// this code.
+AMP.extension('amp-share-tracking', () => {
```

Comment:
Was not going to submit this particular file, hence the `DO NOT SUBMIT`. Let's discuss offline. I'm not seeing a better way to make our tests hermetic, but we can delay this particular change and pursue a bit more manual approach.